### ISSUE TITLE
Website refactor (flask)

### ISSUE THREAD

**Issue by VeckoTheGecko** [OceanParcels/oceanparcels_website#74]
The site currently has data and content intertwined with each other in the HTML making the site a bit difficult to maintain/grow.

I propose to introduce [flask](https://flask.palletsprojects.com/en/2.3.x/) with [Frozen-Flask](https://pythonhosted.org/Frozen-Flask/) to provide tooling to make the site easier to oversee, maintain, and extend.

Main changes:  

- Convert `index.html` and `utrechtteam.html` to templates
  - abstract out the bulk of the data on papers, and team members into `yaml` files. During build, this data is populated into the templates
- Other pages remain untouched
- Markdown support ([flask-markdown](https://pythonhosted.org/Flask-Markdown/)) which compiles md to HTML during build
  - Markdown files can be included directly in the website
  - Markdown can be written directly within the HTML files

Everything according to plan, building the site will create the exact same website files that currently exist. I'm already familiar with this tooling, so this PR shouldn't take me too long to implement.

Blocked by #73

**Comment by erikvansebille** [comment]
Thanks @VeckoTheGecko, for this suggestion for a complete change in the workflow. Maybe you're right that we need a more flashy ('flasky') workflow for the website, but I'm kind of fond of the simpleness of pure html. Call me old-fashioned? In any case, I think this has much lower priority than some of the stuff for the Parcels documentation? Who knows, we might end up with all the Parcels info served through sphinx/parcels-docs anyways?

**Comment by VeckoTheGecko** [comment]
Sounds good. Definitely a lower priority 🙂. Just wanted to also flag it in case there were plans in future to broaden the scope of the website.

**Comment by VeckoTheGecko** [comment]
Original idea still stands for the most part (will likely still go with flask, but also might do something else), but things to note are:

- Using a data file of DOIs to get metadata, and to generate the HTML for list of papers mentioning parcels (also check that it is decent quality)
- Try find the same theme (in a way that any accessibility/most of the HTML is handled by theme maintainers upstream reducing our maintainence)
- Keep the page structure the same (avoid breaking any links)
